src/components/item/sharing/shortLink/ShortLinksRenderer.tsx

```diff
@@ -59,6 +61,11 @@ const ShortLinksRenderer = ({
       const clientHostManager = ClientHostManager.getInstance();
       const url = clientHostManager.getItemAsURL(platform, itemId);
 
+      // not ideal
+      if (platform === Context.Builder) {
+        url.searchParams.set('mode', mode);
+      }
```

Comment:
This implies that the mode used in the link will be the mode currently used. Do we want to allow users to select the mode they want to share ? temporarily this would be enough, but we should think about these design questions.